pkg/validation/seed/validator_test.go

```diff
@@ -415,13 +433,28 @@ func TestValidate(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			sv := &Validator{
+			var obj []runtime.Object
+			for _, c := range tc.existingClusters {
+				obj = append(obj, c)
+			}
+			for _, s := range tc.existingSeeds {
+				obj = append(obj, s)
+			}
```

Comment:
This can be shortened to

```go
obj = append(obj, tc.existingClusters...)
obj = append(obj, tc.existingSeeds...)
```

Reply:
true, why not

Reply:
ah, right. I did it like that in order to have typed slices in the tests:
```
		existingSeeds    []*kubermaticv1.Seed
		existingClusters []*kubermaticv1.Cluster
```
As it's just syntactic sugar and does not add many lines I would rather keep it that way